static/app/views/alerts/incidentRules/ruleNameOwnerForm.tsx

```diff
@@ -1,61 +1,98 @@
-import {PureComponent} from 'react';
+import {Fragment, PureComponent} from 'react';
+import * as React from 'react';
+import styled from '@emotion/styled';
 
 import TeamSelector from 'sentry/components/forms/teamSelector';
+import ListItem from 'sentry/components/list/listItem';
 import {Panel, PanelBody} from 'sentry/components/panels';
 import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
 import {Project, Team} from 'sentry/types';
 import FormField from 'sentry/views/settings/components/forms/formField';
 import TextField from 'sentry/views/settings/components/forms/textField';
 
 type Props = {
   disabled: boolean;
+  hasAlertWizardV3: boolean;
   project: Project;
 };
 
-class RuleNameOwnerForm extends PureComponent<Props> {
-  render() {
+export default class RuleNameOwnerForm extends PureComponent<Props> {
```

Comment:
Should we turn this into a functional component since it doesn't have state?

Reply:
👍